sormas-api/src/main/java/de/symeda/sormas/api/caze/CaseExportDto.java

```diff
@@ -1226,47 +1231,68 @@ public void setNumberOfClinicalVisits(int numberOfClinicalVisits) {
 		this.numberOfClinicalVisits = numberOfClinicalVisits;
 	}
 
-	public void setSampleDateTime1(Date sampleDateTime1) {
-		this.sampleDateTime1 = sampleDateTime1;
+	public CaseExportSampleDto getSample1() {
+		return sample1;
 	}
 
-	public void setSampleLab1(String sampleLab1) {
-		this.sampleLab1 = sampleLab1;
+	public void setSample1(CaseExportSampleDto sample1) {
+		this.sample1 = sample1;
 	}
 
-	public void setSampleResult1(PathogenTestResultType sampleResult1) {
-		this.sampleResult1 = sampleResult1;
+	public CaseExportSampleDto getSample2() {
+		return sample2;
 	}
 
-	public void setSampleDateTime2(Date sampleDateTime2) {
-		this.sampleDateTime2 = sampleDateTime2;
+	public void setSample2(CaseExportSampleDto sample2) {
+		this.sample2 = sample2;
 	}
 
-	public void setSampleLab2(String sampleLab2) {
-		this.sampleLab2 = sampleLab2;
+	public CaseExportSampleDto getSample3() {
+		return sample3;
 	}
 
-	public void setSampleResult2(PathogenTestResultType sampleResult2) {
-		this.sampleResult2 = sampleResult2;
+	public void setSample3(CaseExportSampleDto sample3) {
+		this.sample3 = sample3;
 	}
 
-	public void setSampleDateTime3(Date sampleDateTime3) {
-		this.sampleDateTime3 = sampleDateTime3;
+	public List<CaseExportSampleDto> getOtherSamples() {
+		return otherSamples;
 	}
 
-	public void setSampleLab3(String sampleLab3) {
-		this.sampleLab3 = sampleLab3;
+	public void addOtherSample(CaseExportSampleDto otherSample) {
+		this.otherSamples.add(otherSample);
 	}
 
-	public void setSampleResult3(PathogenTestResultType sampleResult3) {
-		this.sampleResult3 = sampleResult3;
+	public CaseJurisdictionDto getJurisdiction() {
+		return jurisdiction;
 	}
 
-	public void setOtherSamples(String otherSamples) {
-		this.otherSamples = otherSamples;
-	}
+	public static class CaseExportSampleDto implements Serializable {
 
-	public CaseJurisdictionDto getJurisdiction() {
-		return jurisdiction;
+		private Date dateTime;
+		@SensitiveData
+		private String lab;
+		private PathogenTestResultType result;
+
+		public CaseExportSampleDto() {
+		}
+
+		public CaseExportSampleDto(Date dateTime, String lab, PathogenTestResultType result) {
+			this.dateTime = dateTime;
+			this.lab = lab;
+			this.result = result;
+		}
+
+		public String stringFormat() {
```

Comment:
I wouldn't override toString method because because for example if there will be needed an extra function argument and also other generic code will rely on that toString method it will be hard to locate the places where to use the updated toString method. I faced the same issue during the implementation of locale based date formatting.

I can rename to `asString` if you think it's a better name.